components/Pagination/src/pagination.css

```diff
@@ -0,0 +1,133 @@
+  line-height: 1.5;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  width: var(--denhaag-pagination-size);
+
+  font-style: normal;
+  font-weight: var(--denhaag-pagination-font-weight, 500);
+  font-size: var(--denhaag-pagination-font-size);
+}
+
+.denhaag-pagination__link:hover,
+.denhaag-pagination__link--hover {
+  --denhaag-pagination-color: var(--denhaag-pagination-link-hover-color);
+}
+
+.denhaag-pagination__link:focus,
+.denhaag-pagination__link--focus {
+  outline: var(--denhaag-link-focus-outline);
+}
+
+.denhaag-pagination__link:disabled,
+.denhaag-pagination__link--disabled {
+  --denhaag-pagination-color: var(--denhaag-pagination-link-disabled-color);
+  pointer-events: none;
+}
+
+.denhaag-pagination__link--arrow {
+  --denhaag-pagination-size: var(--denhaag-pagination-arrow-size);
+  --denhaag-pagination-font-size: var(--denhaag-pagination-arrow-font-size);
+}
+
+[dir="rtl"] .denhaag-pagination__link--arrow {
+  transform: scaleX(-1);
+}
+
+.denhaag-pagination__link + .denhaag-pagination__links,
+.denhaag-pagination__links + .denhaag-pagination__link {
+  margin-inline-start: var(--denhaag-pagination-margin-inline);
+}
+
+.denhaag-pagination__link:not(:first-child):not(:last-child):not(.denhaag-pagination__link--current):not([rel]) {
+  overflow: hidden;
+  pointer-events: none;
+  text-indent: calc(var(--denhaag-pagination-size) * 2);
+}
+
+.denhaag-pagination__link:not(:first-child):not(:last-child):not(.denhaag-pagination__link--current):not([rel]):before {
+  content: "...";
+  left: calc(50% - 7px);
+  line-height: 0;
+  position: absolute;
+  text-indent: 0;
+  vertical-align: baseline;
+}
+
+.denhaag-pagination__link--current {
+  --denhaag-pagination-color: var(--denhaag-pagination-link-current-color);
+  --denhaag-pagination-background-color: var(--denhaag-pagination-link-current-background-color);
+}
+
+.denhaag-pagination__link--current.denhaag-pagination__link:hover,
+.denhaag-pagination__link--current.denhaag-pagination__link--hover {
+  --denhaag-pagination-color: var(--denhaag-pagination-link-current-hover-color);
+  --denhaag-pagination-background-color: var(--denhaag-pagination-link-current-hover-background-color);
+}
+
+.denhaag-pagination__link--current.denhaag-pagination__link:disabled,
+.denhaag-pagination__link--current.denhaag-pagination__link--disabled {
+  --denhaag-pagination-color: var(--denhaag-pagination-link-current-disabled-color);
+  --denhaag-pagination-background-color: var(--denhaag-pagination-link-current-disabled-background-color);
+}
+
+.denhaag-pagination__link-label {
+  /* Copied from Bootstrap .sr-only */
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin-inline-start: -1px;
+  margin-inline-end: -1px;
+  margin-block-start: -1px;
+  margin-block-end: -1px;
+  overflow: hidden;
+  padding-block-start: 0;
+  padding-block-end: 0;
+  padding-inline-start: 0;
+  padding-inline-end: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+}
```

Comment:
Could you explain what the purpose of this sr-only class is? I'm having trouble understanding when a developer would use it. The navigation can be made accessible without the use of this class. (See comments on bem story about accessibility)